eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -22,12 +11,3 @@ steps:
       arguments: -StorageAccountKey '$(cpp-vcpkg-cache-storage-key)'
     displayName: Set Vcpkg Write-mode Cache
     condition: and(succeeded(), eq(variables['System.TeamProject'], 'internal'))
-
-  - task: PowerShell@2
-    inputs:
-      targetType: filePath
-      filePath: eng/scripts/vcpkg.ps1
```

Comment:
Remove vcpkg.ps1? At this point, I think it's no longer needed.

Reply:
I would let that decision to EngSys team @danieljurek :)

Reply:
If we are no longer using this anywhere I think I will speak for @danieljurek who is out this week and say we should delete this script now.